deploy/docs/SideBySidePrometheus.md

```diff
@@ -86,9 +101,21 @@ helm upgrade --install my-release sumologic/sumologic \
      --set kube-prometheus-stack.prometheus-node-exporter.service.targetPort=9200
 ```
 
+If you are installing with limiting the scope of the interaction of Prometheus Operator:
+
+```bash
+helm upgrade --install my-release sumologic/sumologic \
+     --set sumologic.accessId=<SUMO_ACCESS_ID> \
+     --set sumologic.accessKey=<SUMO_ACCESS_KEY> \
+     --set sumologic.clusterName="<MY_CLUSTER_NAME>" \
+     --set kube-prometheus-stack.prometheusOperator.namespaces.releaseNamespace=true \
+     --set kube-prometheus-stack.prometheus-node-exporter.service.port=9200 \
+     --set kube-prometheus-stack.prometheus-node-exporter.service.targetPort=9200
```

Comment:
Can we make this a yaml configuration snippet instead? I think this encourages some users to just pass all configuration parameters via the command line instead of keeping their own values.yaml.

Reply:
Do you mean the form presented below?

```
helm upgrade --install my-release sumologic/sumologic -f values.yaml
```
values.yaml:
```yaml
sumologic:
    accessId=<SUMO_ACCESS_ID>
    accessKey=<SUMO_ACCESS_KEY>
    clusterName="<MY_CLUSTER_NAME>"
...
```

Reply:
If we want to change the form in which helm command is presented then we need to do it globally in the whole repo and I don't want to make it  in this PR.

Reply:
Allright, I'm ok with that.